docs/en/users/04_Subscriptions.md

```diff
@@ -1,11 +1,93 @@
+ 2. On FreshRSS, click the "**+**" button next to “Subscriptions management”.
+ 3. Paste the URL in “Feed URL” field.
  4. (optional): You can select the category for your feed. By default, it will be in “Uncategorized”.
+ 5. (optional): If the subscription requires credentials, you can enter them in the "HTTP username" and "HTTP password" fields. 
+ 6. (optional): You can get a timeout for the feed request if the feed requires it.
+ 7. (optional): You can choose to ignore SSL certificate errors (such as with self-signed certificates) by setting "Verify SSL security" to "No". This is not recommended, and it is better to either add the root certificate to the FreshRSS server or to fix the SSL certificate problems on the feed hosting server. 
 
-# Import and export
+# Subscription management
+
+The "Subscription management" submenu allows categories and feeds to be configured. Feeds can be moved between categories by drag-and-drop, or in the individual feed's settings. Hovering over a feed/category will cause a gear icon to appear. Clicking the icon will bring up the settings for that item. 
+
+## Category Settings
+
+### Information 
+
+* **Title:** Name of category
+* **Display position:** Defines order of categories. Lower numbers get priority, non-numbered items come last, and equally numbered items will sort by alphabetical order.
+
+### Archiving
+
+If "Purge Policy" has "By default" selected, then the [default purge policy](./05_Configuration.md) is used and the other options are not displayed. Category options will override the default policy, but will not override feed-specific options.
+
+## Feed Settings
+
+### Information
+
+These should be auto-filled when adding a feed, but can be modified after. **Visibility** will define if the feed is displayed in the main feed, only in category feeds, or not at all.
+
+### Archival
+
+This section will let you override the default settings for feed archiving and update frequency.
+
+### Login
+
+Some feeds require a username/password submitted over HTTP. These usually aren't needed for feeds.
+
+### Advanced
+
+#### Retrieve a truncated feed from within FreshRSS
+
+This question comes up regularly, so we'll try to clarify how one can retrieve a truncated RSS feed with FreshRSS. Please note that the process is absolutely not user friendly, but it works. :)
+
+Please be aware that this way you'll generate much more traffic to the originating sites, and they might block you accordingly. FreshRSS performance is also negatively affected, because you'll have to fetch the full article content one by one. So it's a feature to use sparingly!
+
+What's meant by "CSS path of articles on the original site" actually corresponds to the "path" consisting of IDs and classes (which in HTML, matches the id and class attributes) to retrieve only the interesting part that corresponds to the article. Ideally, this path starts with an id (which is unique to the page).
```

Comment:
This is outdated. It now says `Article CSS selector on original website` because, well, that's what it is and you can actually use a search engine with those terms.

Reply:
I hadn't written this section - just moved it to a new place that made more sense. What would you recommend here? I can change the terms or try to re-write the section.

Reply:
I'm not quite sure. Let's just leave it more or less the same for now?
```suggestion
The  "Article CSS selector on original website" corresponds to the "path" consisting of IDs and classes (which in HTML, matches the id and class attributes) to retrieve only the interesting part that corresponds to the article. Ideally, this path starts with an id (which is unique to the page). The basics are explained [here](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Selectors).
```